docs/DappGuide.md

```diff
@@ -3,55 +3,55 @@
 ### Description
 This guide describes the steps to transfer tokens using the Web Interface for the RSK Tokenbridge system. Please refer to the project documentation if you’d like to know more about how this bridge works.
 
-It is possible to test the transfer of tokens between RSK and Kovan networks using the RSK Tokenbridge web interface. This will require access to a Chrome or Chromium web browser and install one of the following wallets [Metamask with custom network](https://github.com/rsksmart/rskj/wiki/Configure-Metamask-to-connect-with-RSK) or [Nifty Wallet](https://chrome.google.com/webstore/detail/nifty-wallet/jbdaocneiiinmjbjlgalhcelgbejmnid). There has been a breaking change in Nifty wallet since version 5.0.0 if you created you account before that time you can just use the RSK Testnet option out of the box, if you created the account after that version you will need to [use a custom network](https://developers.rsk.co/tutorials/resolve-nifty-issue/#add-rsk-as-custom-node) to keep the same address in RSK and Ethereum.
+It is possible to test the transfer of tokens between RSK and Kovan networks using the RSK Tokenbridge web interface. This will require access to a Chrome or Chromium web browser and install one of the following wallets [Metamask with custom network](https://developers.rsk.co/develop/apps/wallets/metamask/) or [Nifty Wallet](https://chrome.google.com/webstore/detail/nifty-wallet/jbdaocneiiinmjbjlgalhcelgbejmnid). The derivation path used in RSK is different from the one used in Ethereum, so you need to [use a custom network](https://developers.rsk.co/tutorials/resolve-nifty-issue/#add-rsk-as-custom-node) to keep the same address in RSK and Ethereum.
 
 ## Steps
 Go to the Dapp [https://tokenbridge.rsk.co/](https://tokenbridge.rsk.co/) for mainnet or [https://testnet.tokenbridge.rsk.co/](https://testnet.tokenbridge.rsk.co/) for testnet.
-First **log in** with your wallet (metamask or nifty), and select the network of your choice, in this case we will use **RSK Testnet**.
+First **log in** with your wallet (metamask or nifty) and select the network of your choice, in this case we will use **RSK Testnet**.
 
 <p align="center">
-  <img src="./images/dapp-image1-1.png" width="800" height="250" />
+  <img src="./images/dapp-image1-1.png"/>
 </p>
 
-If everything is correct, you will see the following:
+If everything is correct, you will see the following screen:
 
 <p align="center">
```

Comment:
```suggestion
```

Reply:
the Html tag is to actually display the image, if you add it as [Bridge Diagram](./docs/images/token-bridge-diagram.png) it just adds it as an hiper link.
So im going to leave it as it is